### DISCUSSION TITLE
Solved

### DISCUSSION THREAD

**Discussion by morteza-ayashi** [Acode-Foundation/Acode#627] [Q&A] [answered]
Solved

**Comment by deadlyjack** [comment, marked as answer]
> ![Screenshot_20230121-183343_Acode.jpg](https://user-images.githubusercontent.com/113019884/213872952-08cfb412-3e45-48bb-90ed-8fdd98f372ee.jpg)
> 
> Hi
> 
> After updating to v1.7.1 build 264 all plugins have been removed

Do you mean installed plugins?
